.github/workflows/test.yml

```diff
@@ -51,9 +51,12 @@ jobs:
       fail-fast: false
       matrix:
         terraform:
-          - '0.12.30'
-          - '0.13.6'
-          - '0.14.5'
+          - '0.12.31'
```

Comment:
Nit: If we switch to using the `hashicorp/setup-terraform@v1` action instead of `TF_ACC_TERRAFORM_VERSION` environment variable we can install the latest patch versions. Refer to [terraform-provider-corner](https://github.com/hashicorp/terraform-provider-corner/blob/507647c80582ff6f4e4d3d81536db5906419e037/.github/workflows/test.yml#L48-L66) for an example. 😄

Reply:
yeah, actually I have seen your PR yesterday and I think we should do the switch.

Having the flexibility to test against the latest patch and not having to manually maintain this list is the best way.

Will update this PR.

PS One last review before holidays? ;)